spec/swagger/diff/diff_spec.rb

```diff
@@ -29,7 +29,8 @@
                                  'missing request param: obj/thing (type: integer)',
                                  'missing request param: str (type: string)'
                                 ],
-               'put /b/{}' => ['new required request param: extra'] },
+               'put /b/{}' => ['new required request param: extra'],
+               'post /c/' => ['new required request param: existing/b'] },
```

Comment:
Given the title/description of this PR, I would have expected these spec diffs to include you removing some line(s) that used to false-positive warn about "new required params", but this diff is all additions.  Am I missing something?